go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7
 	github.com/avast/retry-go v3.0.0+incompatible
+	github.com/deepmap/oapi-codegen v1.8.2 // indirect
```

Comment:
Seems strange that for such a small PR you get all these dependencies added 🤔 

Maybe check that you have the latest golang version and run go mod tidy.